strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/ProxyConfiguration.java

```diff
@@ -39,8 +39,7 @@
      */
     @XmlAttribute
     private String type;
-
-    @XmlElement(name = "host")
```

Comment:
Not sure why this was removed...? (I thought that `non-proxy-host` was too long, so I thought of just using `host` instead. Is this causing a problem?